Stylesheets/components/utility.css

```diff
@@ -0,0 +1,36 @@
+@import url(/Stylesheets/main.css);
+
+.heading-text,
+.para-text {
+  padding: 10px;
+}
+.body-cp-lg {
+  font-size: 2rem;
+  line-height: 2.5rem;
+}
+.body-cp-rg {
+  font-size: 1.5rem;
+  line-height: 2.5rem;
+}
+.body-cp-md {
+  font-size: 1rem;
+  line-height: 2.5rem;
+}
+.body-cp-sm {
+  font-size: 0.75rem;
+  line-height: 2.5rem;
```

Comment:
body-cp-md class is not sound good 
or you can set font-l(font lasge) , font-md(font-meditext) like this so you can understand these class by name

Reply:
yes u r right but i think  we can use shortform , it is easy to name and maintain.

Reply:
text-lg
text-md
text-sm
these are small and shortform

Reply:
okay done